.konflux/operator/rpms.in.yaml

```diff
@@ -0,0 +1,21 @@
+contentOrigin:
+  # Define at least one source of packages, but you can have as many as you want.
+  repofiles:
+     - ./redhat.repo
+
+packages:
+  # list of rpm names to resolve
+  - hwdata
+
+arches:
+  # The list of architectures can be set in the config file. Any `--arch` option set
+  # on the command line will override this list.
+  - x86_64
+
+context:
+  # Alternative to setting command line options. Usually you will only want
+  # to include one of these options, with the exception of `flatpak` that
+  # can be combined with `image` and `containerfile`
+  containerfile:
+    file: konflux.Dockerfile
+    imagePattern: registry.redhat.io/rhel9-4-els/rhel
```

Comment:
I don't see this field amongst the allowed ones specified in the readme here:
https://github.com/konflux-ci/rpm-lockfile-prototype?tab=readme-ov-file#whats-the-input_file
can you share a link where this is declared and how it is used? 
I would guess this the prefix of the final base image to use in the dockerfile while building the image, but Id rather confirm this from the docs.

Reply:
on the same link you sent you can see it just a bit above this section: https://github.com/konflux-ci/rpm-lockfile-prototype?tab=readme-ov-file#what-does-this-do   (please also note that this repo is a helper tool and does not guarantee 100% stability) 

I also used this reference from LVM: https://github.com/openshift/lvm-operator/blob/main/release/operator/rpms.in.yaml#L31